core/src/main/java/com/linecorp/armeria/common/RequestContextAwareCompletableFuture.java

```diff
@@ -251,7 +254,7 @@
     }
 
     public CompletionStage<T> minimalCompletionStage() {
-        return this.uniAsMinimalStage();
+        return ctx.makeContextAware(new RequestContextAwareMinimalStage<>(this));
```

Comment:
I guess we do not need to make `RequestContextAwareMinimalStage` context-aware because `this` is already context-aware.